license/license.go

```diff
@@ -51,7 +55,9 @@ var generatedHeaders = []string{
 
 func init() {
 	Analyzer.Flags.StringVar(&ignoreFilesPattern, "ignore", "", "Comma separated list of files to ignore")
+	Analyzer.Flags.StringVar(&licenseYear, "year", "2015", "Year to use in license header (2015-present)")
```

Comment:
This is only ever really used as an integer, so could we parse it as one rather than a string that is later converted? I think this would also mean error checking happens automatically